bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -396,12 +396,12 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
                     // probably a ExtJS 6.0 bug. withou this, dropdowns not working anymore if plugin is enabled
```

Comment:
can you fix the spelling error becasue you touch it anyway :-)

Reply:
done